lib/xboxrt/libc_extensions/direct.h

```diff
@@ -3,3 +3,11 @@
 // SPDX-FileCopyrightText: 2020 Jannik Vogel
 
 // Part of Microsoft CRT
+
+int _chdir(char* path);
```

Comment:
The parameter should be `const char *path` (applies to the function definition as well)